web/src/components/config_render/ConfigGroup.jsx

```diff
@@ -15,31 +15,28 @@ import { setOrder } from "./ConfigUtil";
 import { ConfigWrapper, ConfigItems } from "./ConfigComponents";
 import Icon from "../Icon";
 
-export default class ConfigGroup extends React.Component {
-  constructor() {
-    super();
-    this.markdownNode = React.createRef();
-  }
+const ConfigGroup = (props) => {
+  const markdownNode = React.createRef();
```

Comment:
When I first worked on this ticket I thought I would put the observer in this file and use a hook. I converted this file to functional component and changed my mind about the implementation.